workshops/ARC.md

```diff
@@ -0,0 +1,42 @@
+# Workshops Arc
+
+This document outlines our curriculum arc. In many ways, this is the syllabus
+for Hack Club.
+
+Hack Clubs are after-school high school clubs where people build things
+together. They can almost be thought of as never-ending hackathons, where club
+members come together twice a week (generally for ~1.5 hours) and work on
+projects together. Throughout the school year, Hack Clubs go (and sometimes
+host) hackathons, work on larger group projects, and---most importantly---build
+a thriving community of hackers at their high schools.
+
+While the goal with Hack Clubs is for most meetings to be free-form time where
+hackers are working on projects together, we're actively developing a series of
+open-source workshops to bring members from no experience to the point where
+they can hack on projects.
+
+Hackers use the following tools for workshops:
+
+- [Cloud9](https://c9.io/) - development environment
+- [GitHub](https://github.com/) - code host
+  - All Hack Club development goes into a
+    [GitHub Pages](https://pages.github.com/) repository called
+    `[username].github.io` (replacing `[username]` with their GitHub username)
+  - Each project done in Hack Club (so at least each workshop) goes into a
+    sub-directory called `project_name` in the main `[username].github.io`
+    project. The hackers should be regularly committing and pushing. For
+    example, if I were doing the Portfolio workshop, I'd put my work in
+    `[username].github.io/portfolio/`.
+
+Below is the initial plan for the Hack Club workshop progression.
+
+| Week # | Meeting # | Concept                       | Workshop (blank if not created yet) |
+|--------+-----------+-------------------------------+-------------------------------------|
```

Comment:
Currently the Markdown tables are not rendering correctly:

> 
![image](https://cloud.githubusercontent.com/assets/654941/12182607/0284a1de-b53e-11e5-8d14-ba223cabdbe2.png)

I believe you want to be using `|` instead of `+` to get this to render correctly, like so:

| Week # | Meeting # | Concept                       | Workshop (blank if not created yet) |
|--------|-----------|-------------------------------|-------------------------------------|
|      1 |         1 | First code ever (HTML & CSS)  | [Portfolio][portfolio]              |
|      2 |         3 | Hack Club onboarding          |                                     |
|      2 |         3 | First JavaScript ever         | [Twilio][twilio]                    |
|      3 |         5 | Solidifying JavaScript basics |                                     |
|      4 |         7 | Loops & control structures    |                                     |